co-authors-plus-social-pack.php

```diff
@@ -197,6 +199,46 @@ public function filter_coauthors_guest_author_fields( $fields_to_return, $groups
 		return apply_filters( 'coauthors_social_fields', $fields_to_return );
 	}
 
+	/**
+	 * Hook into Co Authors Plus's filter coauthors_guest_author_personal_export_data to extra fields
+	 * to exported personal data
+	 *
+	 * @param  array $data      Data to be exported
+	 * @param  array $author_id The guest author ID
+	 * @return array            The extra data to be exported
+	 */
+	function filter_export_extra_data( $data, $author_id ) {
+		$fields = $this->coauthors_plus->guest_authors->get_guest_author_fields( 'social' );
+		$ignore_fields = array(
+			'enable_twitter_via',
+			'enable_twitter_related',
+		);
+
+		if ( ! is_array( $data ) ) {
+			$data = [];
+		}
+
+		foreach( $fields as $field ) {
+			if ( in_array( $field['key'], $ignore_fields ) ) {
+				continue;
+			}
+
+			$key = $this->coauthors_plus->guest_authors->get_post_meta_key( $field['key'] );
+			$value = get_post_meta( $author_id, $key, true );
+
+			if ( empty( $value ) ) {
+				continue;
+			}
+
+			$data[] = array(
+				'name'	=> __( 'Guest Author', 'co-authors-plus' ) . ' ' . $field['label'],
```

Comment:
We can remove the `Guest Author` prefix here.

Reply:
Done